hadoop-hdds/server-scm/src/main/resources/webapps/scm/scm.js

```diff
@@ -81,6 +91,18 @@
                     $scope.totalItems = nodeStatusCopy.length;
                     $scope.lastIndex = Math.ceil(nodeStatusCopy.length / $scope.RecordsToDisplay);
                     $scope.nodeStatus = nodeStatusCopy.slice(0, $scope.RecordsToDisplay);
+
+                    ctrl.nodemanagermetrics.NodeStatistics.forEach(function(obj) {
```

Comment:
Just fyi, can de-structure this as well like you did for map().
Since obj = { key: abc, value: xyz }, we can destructure it as:
{key, value} = obj

So the function could also be written as:
```
ctrl.nodemanagermetrics.NodeStatistics.forEach(({key, value}) => {
  ....
});
```
But your current change is good as well